src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2460,6 +2487,14 @@ void processContextElement
       searchContextProviders(tenant, servicePathV, *enP, ceP->contextAttributeVector, cerP);
       cerP->statusCode.fill(SccOk);
       responseP->contextElementResponseVector.push_back(cerP);
+
+      //
+      // If no context providers found, then the UPDATE was simply for a non-found entity and an error should be returned
+      //
+      if (forwardsPending(responseP) == false)
```

Comment:
Not so sure about this.
That way, mongoBackend returns OK when really we are at case of a NOT FOUND ...
Are the unittests correct?
I would say, lets take a good look at the unittests. If they are 100% ok the way they were, then lets go back.
But if not, then lets keep this the way it is.